specification/app/resource-manager/Microsoft.App/preview/2023-08-01-preview/examples/Patches_Apply.json

```diff
@@ -0,0 +1,88 @@
+{
+  "operationId": "Patches_Apply",
+  "title": "Patches_Apply_0",
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "resourceGroupName": "rg",
+    "builderName": "testBuilder",
+    "patchName": "testPatch-25fe4b",
+    "api-version": "2023-08-01-preview"
+  },
```

Comment:
So now I'm curious, aren't any other parameters required? How did it know which patches to apply?

Reply:
the patch is applied by specifying patch name. "patchDetails" explains describes what operation is going to be done when user applies the patch. All the params are necessary

Reply:
So is there any practical barrier to changing this to use PUT then?

Reply:
i don't think there's barrier tho. But I think put method is more used when creating a new resource rather than interacting with an existing one? But if interacting with an existing one could use put, I can definitely change that in the next commit!